packages/coinstac-docker-manager/test/index.test.js

```diff
@@ -0,0 +1,348 @@
+const test = require('ava');
+const sinon = require('sinon');
+const portscanner = require('portscanner');
+const winston = require('winston');
+const { createReadStream, unlink } = require('fs');
+const unzip = require('unzipper');
+const resolvePath = require('path').resolve;
+const wsServer = require('../../coinstac-images/coinstac-base/server-ws2');
+const httpServer = require('../../coinstac-images/coinstac-base/server-http');
```

Comment:
I think this is a good way to include the files, but why have the mock servers too?